apps/front-end/src/components/map/mapSlice.ts

```diff
@@ -1,12 +1,13 @@
 import { createSelector } from "@reduxjs/toolkit";
 import { z } from "zod";
+
+import { notNullish, schemas } from "@mykomap/common";
 import { createAppSlice } from "../../app/createAppSlice";
 import { getDatasetLocations } from "../../services";
-import { notNullish, schemas } from "@mykomap/common";
+import { getUrlSearchParam } from "../../utils/window-utils";
```

Comment:
I try to keep imports from external libraries at the top of the list, but it's just a personal preference, probably a bit of OCD. Will try to avoid in the future though, if it's making reviews difficult